### PR TITLE
docs: simplify instructions

### PR DESCRIPTION
**Pull request by KobeW50**
Im a programmer now :teef:

### CODE REVIEW

File: auto-cli.bat [hunk 1]
```diff
@@ -252,7 +252,7 @@ if exist ..\revanced-cli-input\patches.jar (
 	pause
 	"%JDK%" -jar "%CLI%" list-patches -dopv "%PATCHES%"
 ) else (
-	echo  You are using official ReVanced patches. Please look up the patches names at revanced.app/patches.
+	echo  You are using official ReVanced patches. [92m Please look up the patch names at https://revanced.app/patches.[0m
```
Comment (KobeW50) [inline auto-cli.bat]: i have no clue how to highlight stuff but this seemed the way to do it. pls make sure I didn't mess up

File: auto-cli.bat [hunk 2]
```diff
@@ -252,7 +252,7 @@ if exist ..\revanced-cli-input\patches.jar (
 	pause
 	"%JDK%" -jar "%CLI%" list-patches -dopv "%PATCHES%"
 ) else (
-	echo  You are using official ReVanced patches. Please look up the patches names at revanced.app/patches.
+	echo  You are using official ReVanced patches. [93m Please look up the patch names at https://revanced.app/patches.[0m
```
Comment (KobeW50) [inline auto-cli.bat]: idk how to highlight but this seemed the way to do it

File: auto-cli.bat [hunk 3]
```diff
@@ -252,7 +252,7 @@ if exist ..\revanced-cli-input\patches.jar (
 	pause
 	"%JDK%" -jar "%CLI%" list-patches -dopv "%PATCHES%"
 ) else (
-	echo  You are using official ReVanced patches. Please look up the patches names at revanced.app/patches.
+	echo  You are using official ReVanced patches. [93m Please look up the patch names and capitalizations at https://revanced.app/patches.[0m
```
Comment (KobeW50) [inline auto-cli.bat:255]: i hope i highlighted correctly.

i honestly think this should be stressed more, or somehow placed in the paragraph below

**Comment by taku-nm** [comment]
Oh look first PR :steamhappy:

**Pull request merged**